example/webpack.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
     ],
   },
   devServer: {
-    contentBase: path.resolve(__dirname, '.'),
```

Comment:
Whats the purpose for this change? It appears that this change breaks `npm run start`